service/hub/README.md

```diff
@@ -1,9 +1,27 @@
-### Building local image
-
-```
-mkdir -p /var/tmp/go/src/github.com/cesanta
-cd /var/tmp/go/src/github.com/cesanta
-git clone https://github.com/cesanta/docker_auth.git
-cd docker_auth/auth_server
-make docker-build
-```
+# service-hub
+
+Hub service is the auth service for sealos registry.
+
+## Description
```

Comment:
Add intro how to real auth:
1. How to check user exists?
2. How to checkout user have control?
3. How to combine `kubeconfig` with docker registry's match?

Reply:
added.